pycparser/ply/yacc.py

```diff
@@ -32,7 +32,7 @@
 # -----------------------------------------------------------------------------
 #
 # This implements an LR parser that is constructed from grammar rules defined
-# as Python functions. The grammer is specified by supplying the BNF inside
```

Comment:
Please revert all changes in the `ply/` directory - it's a vendored dependency and I don't want any changes in there